tests/models/bark/test_modeling_bark.py

```diff
@@ -137,14 +135,15 @@ def get_config(self):
     def get_pipeline_config(self):
         config = self.get_config()
         config.vocab_size = 300
+        config.output_vocab_size = 300
```

Comment:
I don't know why, but if they are different values, there is an issue when trying to load a model after saving it. Probably something to look in the future (maybe just a false alarm)